buildroot-external/patches/occu/0153-WebUI-DynamicResizing.patch

```diff
@@ -0,0 +1,142 @@
+@@ -80,7 +59,7 @@
+       </tr>
+       <tr>
+         <td>
+-          <div id="content" style="background-repeat:no-repeat;background-position:480px center;">
++          <div id="content">
+             <table id="login_content" style="width: 200px;">
+               <tr>
+                 <!-- Liste der Benutzer -->
+@@ -168,9 +147,6 @@
+ 
+ 
+     <script type="text/javascript">
+-      Event.observe(window, "resize", function(event) { recalcDivs(); });
+-      recalcDivs();
+-
+       gotoLoginPage = function() {
+         location.href = "/login.htm";
+       };
+--- occu/WebUI/www/rega/logout.htm.orig
++++ occu/WebUI/www/rega/logout.htm
+@@ -9,30 +9,9 @@
+     <script type="text/javascript" src="/webui/js/extern/prototype.js"></script>
+     <script type="text/javascript" src="/webui/js/lang/loadTextResource.js"></script>
+     <script type="text/javascript" src="/webui/js/lang/translate.js"></script>
+-
+-    <script type="text/javascript">
+-      recalcDivs = function()
+-      {
+-        var wHeight = 0;
+-        if (window.innerHeight) {
+-          wHeight = window.innerHeight;
+-        }
+-        else if (document.documentElement.offsetHeight) {
+-          wHeight = document.documentElement.offsetHeight;
+-        }
+-
+-        var wHeaderHeight = 72;
+-        var wFooterHeight = 32;
+-        var wContentHeight = wHeight - wHeaderHeight - wFooterHeight - 27;
+-
+-        $("navbar").setStyle({height: wHeaderHeight + "px"});
+-        $("content").setStyle({height: wContentHeight + "px"});
+-        $("footer").setStyle({height: wFooterHeight + "px"});
+-      };
+-    </script>
+   </head>
+   <body>
+-    <table style="width: 100%; height: 100%;" cellpadding="0" cellspacing="0" id="maintable" class="j_translate">
++    <table cellpadding="0" cellspacing="0" id="maintable" class="j_translate">
+       <tr>
+         <td id="navbar">
+           <table cellpadding="0" cellspacing="0">
+@@ -47,7 +26,7 @@
+       </tr>
+       <tr>
+         <td>
+-          <div id="content" style="background-repeat:no-repeat;background-position:480px center;">
++          <div id="content">
+             <table id="logout_content" cellspacing="8">
+               <tr>
+                 <td style="vertical-align: top;">
+@@ -66,9 +45,6 @@
+     </table>
+ 
+     <script type="text/javascript">
+-      Event.observe(window, "resize", function(event) { recalcDivs(); });
+-      recalcDivs();
+-
+       gotoLoginPage = function() {
+         location.href = "/login.htm";
+       };
+--- occu/WebUI/www/webui/style.css.orig
++++ occu/WebUI/www/webui/style.css
+@@ -134,9 +134,23 @@
+  * * *   Element IDs                     * * *
+  * * * * * * * * * * * * * * * * * * * * * * */
+ 
++/* Login- und Logout-Fenster */
++
++#maintable {
++  width: 100vw;
++  height: 100vh;
++}
++
++#maintable #content {
++  height: 100%;
++  background-repeat: no-repeat;
++  background-position: 480px center;
++}
++
+ /* Navigation-Leiste */
```

Comment:
same here

Reply:
fixed, but `/* Navigation-Leiste */` is an original comment.

Reply:
You are right! then it is fine to keep it!